tests/cli.test.js

```diff
@@ -0,0 +1,607 @@
+      const modifiedBuffer = fs.readFileSync(path.join(tempDir, `${fileBasename}.svg`));
+      const isBufferEquals = origBuffer.equals(modifiedBuffer);
+
+      expect(isBufferEquals).toBeFalsy();
+      expectStringContains(modifiedBuffer.toString(), 'fill="none"');
+    });
+
+    test('Should containing “stroke="none"”', () => {
+      const fileBasename = 'svg-not-optimized';
+      const stdout = runCliWithParams(`${workDir}${fileBasename}.svg`);
+
+      expectStringContains(stdout, `${workDir}svg-not-optimized.svg`);
+
+      const origBuffer = fs.readFileSync(path.join(imagesDir, `${fileBasename}.svg`));
+      const modifiedBuffer = fs.readFileSync(path.join(tempDir, `${fileBasename}.svg`));
+      const isBufferEquals = origBuffer.equals(modifiedBuffer);
+
+      expect(isBufferEquals).toBeFalsy();
+      expectStringContains(modifiedBuffer.toString(), 'stroke="none"');
+    });
+  });
+
+  describe('Converting to AVIF (--avif)', () => {
+    describe('Lossy', () => {
+      test('JPEG should be converted', () => {
+        const file = 'jpeg-not-optimized.jpeg';
+        const stdout = runCliWithParams(`--avif ${workDir}${file}`);
+
+        expectFileRatio({ stdout, file, maxRatio: 90, minRatio: 85, outputExt: 'avif' });
+        expectFileNotModified(file);
+      });
+
+      test('PNG should be converted', () => {
+        const file = 'png-not-optimized.png';
+        const stdout = runCliWithParams(`--avif ${workDir}${file}`);
+
+        expectFileRatio({ stdout, file, maxRatio: 95, minRatio: 90, outputExt: 'avif' });
+        expectFileNotModified(file);
+      });
+
+      test('GIF should be converted', () => {
+        const file = 'gif-not-optimized.gif';
+        const stdout = runCliWithParams(`--avif ${workDir}${file}`);
+
+        expectFileRatio({ stdout, file, maxRatio: 95, minRatio: 90, outputExt: 'avif' });
+        expectFileNotModified(file);
+      });
+
+      test('Files should not be converted if ratio <= 0', () => {
+        const fileBasename = 'jpeg-one-pixel';
+        const stdout = runCliWithParams(`--avif ${workDir}${fileBasename}.jpg`);
+
+        expectStringContains(stdout, 'Converting 1 image (lossy)...');
+        expectStringContains(stdout, 'Done!');
+        expectFileNotModified(`${fileBasename}.jpg`);
+        expectFileNotExists(`${fileBasename}.avif`);
+      });
+
+      test('Files in provided directory should be converted', () => {
+        const fileBasename = 'png-not-optimized';
+        const stdout = runCliWithParams(`--avif ${workDir}`);
+        const stdoutRatio = grepTotalRatio(stdout);
+
+        expectStringContains(stdout, 'Converting 6 images (lossy)...');
+        expectRatio(stdoutRatio, 85, 95);
+        expectFileNotModified(`${fileBasename}.png`);
+        expectFileExists(`${fileBasename}.avif`);
+      });
+    });
+
+    describe('Lossless (--lossless)', () => {
+      // TODO: In lossless mode JPEG file size is always smaller than AVIF
```

Comment:
@343dev, huh?

Reply:
I'll fix it later. I need to do some research.